src/Form/index.tsx

```diff
@@ -1557,10 +1550,18 @@ function Form({
   );
 }
 
+// normal <Form /> component is exported with just `props: Props`, so need to
+// export this for use in renderAt. Not needed for FormWithRouter since this is
+// all in the same file
+export function InternalForm(props: Props & InternalProps) {
```

Comment:
the renderAt form needs router functionality no? for browser nav and stuff

Reply:
true, adjusted InternalForm and FormWithRouter so that internal form uses the router without us exposing the InternalProps to external users